packages/sanity/src/structure/panes/document/documentPanel/header/DocumentHeaderTitle.tsx

```diff
@@ -41,23 +41,22 @@ export function DocumentHeaderTitle(): ReactElement {
   }
 
   return (
-    <Flex flex={1} gap={0}>
-      <Flex flex="none" gap={3} padding={2}>
-        <Text size={1}>{createElement(schemaType?.options?.icon || DocumentIcon)}</Text>
-        <Text
-          muted={!value?.title}
-          size={1}
-          textOverflow="ellipsis"
-          weight={value?.title ? 'semibold' : undefined}
-        >
-          {value?.title || (
-            <span style={{color: 'var(--card-muted-fg-color)'}}>
-              {t('panes.document-header-title.untitled.text')}
-            </span>
-          )}
-        </Text>
-      </Flex>
-
+    <Flex flex="none" gap={3} paddingX={2} style={{flex: 1, alignItems: 'center'}}>
```

Comment:
`flex="none"` with `style={{flex:1 …` made me look twice.

Is `flex={1} align="center"` not enough? If we need the `style`, maybe it can be assigned to a constant and hoisted to the outer scope?

Reply:
Oh yeah this works!
For some reason I completely forgot that the Sanity UI items of course had these as props 🤦 what a brain fart, sorry about that!

Pushed a fix!